central/usage/csv/csv_handler.go

```diff
@@ -0,0 +1,74 @@
+package usagecsv
+
+import (
+	"net/http"
+	"net/url"
+	"time"
+
+	"github.com/gogo/protobuf/types"
+	"github.com/pkg/errors"
+	"github.com/stackrox/rox/central/usage/datastore"
+	"github.com/stackrox/rox/pkg/protoconv"
+)
+
+var bom = ([]byte)("\uFEFF")
+
+func writeError(w http.ResponseWriter, code int, err error, description string) {
+	w.Header().Set("Content-Type", "text/plain")
+	w.WriteHeader(code)
+	_, _ = w.Write([]byte(errors.Wrap(err, description).Error()))
+}
+
+// CSVHandler returns an HTTP handler function that serves usage data as CSV.
+func CSVHandler(ds datastore.DataStore) http.HandlerFunc {
+	return func(w http.ResponseWriter, r *http.Request) {
+		from, to, err := parseRequest(r)
+		if err != nil {
+			writeError(w, http.StatusBadRequest, err, "bad CSV usage metrics request")
+			return
+		}
+
+		metrics, err := ds.Get(r.Context(), from, to)
```

Comment:
Here we `GetAll` how about having a `Walk` with function that will stream this out to CSV.